### PR TITLE
Add refresh option to recent transaction activity.

### PR DESCRIPTION
**Pull request by Ishadijcks**
Fancy pulldown to refresh added to recent transaction activity.

### CODE REVIEW

File: BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/activities/RecentTransactionsActivity.java [hunk 1]
```diff
@@ -44,11 +46,22 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        SwipeRefreshLayout = (SwipeRefreshLayout) findViewById(R.id.activity_recent_transactions_swipe_refresh_layout);
+        SwipeRefreshLayout.setOnRefreshListener(new SwipeRefreshLayout.OnRefreshListener() {
+            @Override
+            public void onRefresh() {
+                retrieveRecentTransactions();
+            }
+
+        });
+
+
```
Comment (joskuijpers) [inline BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/activities/RecentTransactionsActivity.java]: Too many newlines

File: BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/activities/RecentTransactionsActivity.java [hunk 2]
```diff
@@ -44,11 +46,22 @@ protected void onCreate(Bundle savedInstanceState) {
...
-        if(getSupportActionBar() != null) {
+        if (getSupportActionBar() != null) {
             Log.d("GUI", "onCreate: Actionbar found");
             getSupportActionBar().setDisplayHomeAsUpEnabled(true);
             getSupportActionBar().setDisplayShowHomeEnabled(true);
-        };
+        }
+
+        SwipeRefreshLayout = (SwipeRefreshLayout) findViewById(R.id.activity_recent_transactions_swipe_refresh_layout);
+        SwipeRefreshLayout.setOnRefreshListener(new SwipeRefreshLayout.OnRefreshListener() {
```
Comment (joskuijpers) [inline BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/activities/RecentTransactionsActivity.java]: Try to use a lambda here.

```java
SwipeRefreshLayout.setOnRefreshListener(() -> retrieveRecentTransactions())
```

**Review by joskuijpers — CHANGES_REQUESTED** [review]

File: BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/activities/RecentTransactionsActivity.java (hunk 2 above)
Comment (Ishadijcks) [reply BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/activities/RecentTransactionsActivity.java]: Done

File: BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/activities/RecentTransactionsActivity.java (hunk 1 above)
Comment (Ishadijcks) [reply BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/activities/RecentTransactionsActivity.java]: Done

**Review by joskuijpers — APPROVED** [review]

**Pull request merged**